charts/testrunnerdashboard/templates/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+{{- $imageTag := $.Values.imageConfig.tag | default $.Values.global.applicationVersion -}}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $.Values.name }}-deployment
+  labels: {{ include "testrunnerdashboard.labels" . | nindent 4 }}
+spec:
+  revisionHistoryLimit: 0
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+  replicas: {{ $.Values.replicas }}
+  selector:
+    matchLabels:
+      deployment: {{ $.Values.name }}-deployment
+  template:
+    metadata:
+      labels:
+        deployment: {{ $.Values.name }}-deployment
+{{ include "testrunnerdashboard.labels" . | indent 8 }}
+    spec:
+      serviceAccountName: {{ .Values.name }}-service-ksa
+      containers:
+      - name: {{ $.Values.name }}-app
+        image: {{ $.Values.imageConfig.repository }}:{{ $imageTag }}
+        imagePullPolicy: {{ $.Values.imageConfig.imagePullPolicy }}
+        ports:
+        - name: main
+          containerPort: 8050
+          protocol: TCP
+        # Note: These readiness settings only apply to Kubernetes' internal load
+        # balancing mechanism -- there's a separate health check setting for
+        # the Ingress / GCP load balancer in the Ingress's backendConfig
+        {{- if .Values.probes.readiness.enabled }}
+        readinessProbe:
+          {{- toYaml .Values.probes.readiness.spec | nindent 10 }}
+        {{- end }}
+        {{- if .Values.probes.liveness.enabled }}
+        livenessProbe:
+          {{- toYaml .Values.probes.liveness.spec | nindent 10 }}
+        {{- end }}
+        {{- if $.Values.probes.startup.enabled }}
+        startupProbe:
+          {{- toYaml $.Values.probes.startup.spec | nindent 10 }}
+        {{- end }}
+        command: ["python",
+                  "dashboard.py"]
```

Comment:
Nit, let's avoid multiline json lists in yaml. Either use this:

```yaml
	command: ["python", "dashboard.py"]
```

or this:

```yaml
        command:
	  - python
	  - dashboard.py
```

Reply:
Updated to yaml list syntax.